packages/cfd/src/Containers/cfd-password-modal/cfd-password-form/cfd-password-form.tsx

```diff
@@ -0,0 +1,297 @@
+                                        />
+                                    )}
+                                </Text>
+                            )}
+                            {/* TODO: Check if do we have to add 'PasswordMeter' in production aswell. */}
+                            <div className='input-element'>
+                                <PasswordMeter
+                                    input={values.password}
+                                    has_error={!!(touched.password && errors.password)}
+                                    custom_feedback_messages={getErrorMessages().password_warnings}
+                                >
+                                    <PasswordInput
+                                        autoComplete='new-password'
+                                        label={
+                                            <Localize
+                                                i18n_default_text='{{platform}} password'
+                                                values={{
+                                                    platform: getCFDPlatformLabel(platform),
+                                                }}
+                                            />
+                                        }
+                                        error={
+                                            (touched.password && errors.password) ||
+                                            (values.password.length === 0 ? error_message : '')
+                                        }
+                                        name='password'
+                                        value={values.password}
+                                        onBlur={handleBlur}
+                                        onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                            handlePasswordInputChange(e, handleChange, validateForm, setFieldTouched);
+                                        }}
+                                        data_testId={`dt_${platform}_password`}
+                                    />
+                                </PasswordMeter>
+                            </div>
+                            {is_real_financial_stp && (
+                                <div className='dc-modal__container_cfd-password-modal__description'>
+                                    <Localize
+                                        i18n_default_text='Your MT5 Financial STP account will be opened through {{legal_entity_name}}. All trading in this account is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA). None of your other accounts, including your Deriv account, is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA).'
+                                        values={{
+                                            legal_entity_name: getLegalEntityName('fx'),
```

Comment:
@aizad-deriv just a question, why did you hardcode this 'fx' value? Is it because scope of this card only one jurisdiction, am I right?

Reply:
@sergei-deriv I'm also not too sure on this one. It was from the previous code. Maybe it is because it has only one jurisdiction.